iOS/Flows/Room/NoticeFooterView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  NoticeFooterView.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 4/19/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class NoticeFooterView: UICollectionReusableView {
+    
+    let pageIndicator = UIPageControl()
+    var cancellables = Set<AnyCancellable>()
+            
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        self.initializeViews()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        self.initializeViews()
+    }
+    
+    deinit {
+        self.cancellables.forEach { (cancellable) in
+            cancellable.cancel()
+        }
+    }
+    
+    func initializeViews() {
+        
+        self.addSubview(self.pageIndicator)
+        self.pageIndicator.currentPageIndicatorTintColor = ThemeColor.B5.color
+        self.pageIndicator.pageIndicatorTintColor = ThemeColor.B2.color
+        self.pageIndicator.hidesForSinglePage = true
+        
+        NotificationCenter.default.publisher(for: .onNoticeIndexChanged)
```

Comment:
This was fun. Notification Center for the win when wanting to pass data into a reusable view from a layout init. works beautifully

Reply:
This solves a problem of wanting full sized content, but not knowing if there was additional notices to scroll too.